OpenSim/Simulation/Model/PhysicalFrame.cpp

```diff
@@ -339,19 +353,22 @@ void PhysicalFrame::convertDisplayGeometryToGeometryXML(SimTK::Xml::Element& bod
     }
 }
 
-// This private method creates a frame in the owner model document with passed in name and content relative to bodyName
-void PhysicalFrame::createFrameForXform(const SimTK::Xml::element_iterator& frameSetIter, const std::string& frameName, const SimTK::Vec6& localXform, const std::string& bodyName)
+void PhysicalFrame::createFrameForXform(
+        const SimTK::Xml::element_iterator& ownerIter,
+        const std::string& frameName, const SimTK::Vec6& localXform,
+        const std::string& parentConnecteeName)
 {
     SimTK::Xml::Element frameNode("PhysicalOffsetFrame");
     frameNode.setAttributeValue("name", frameName);
     stringstream ss;
     ss << localXform[3] << " " << localXform[4] << " " << localXform[5];
     SimTK::Xml::Element translationNode("translation", ss.str());
-    ss.clear();
+    ss = stringstream();
```

Comment:
I don't think we have to wipeout the existing stream, just reset it to empty string, e.g. `ss.str("");` and then `ss.clear();`